src/python/pants/backend/native/subsystems/xcode_cli_tools.py

```diff
@@ -140,23 +140,27 @@ def linker(self):
     return Linker(
       path_entries=self.path_entries(),
       exe_filename='ld',
-      library_dirs=[])
+      library_dirs=[],
+      linking_library_dirs=[],
+      extra_args=['-mmacosx-version-min=10.11'])
```

Comment:
The 10.11 seems arbitrarty / tied to Pants current min version support. Does this deserve to be lifted out to a constant used to generate all such flags?

Reply:
It does! And it was, in `MIN_OSX_VERSION_ARG` from b2a750bf6.